src/Main.cc

```diff
@@ -333,6 +333,57 @@ static int tclReadlineInit(Tcl_Interp* interp)
 }
 #endif
 
+namespace {
+// A fallback from the hardcoded TCLRL_LIBRARY path.
+std::string findPathToTclreadlineInit(Tcl_Interp* interp)
+{
+  std::string path(TCLRL_LIBRARY "/tclreadlineInit.tcl");
+  if (is_regular_file(path.c_str())) {
+    return path;
+  }
+
+  // TL;DR it is possible to run the OpenROAD binary from within the
+  // official Docker image on a different distribution than the
+  // distribution within the Docker image.
```

Comment:
Why do you want to do this?

Reply:
it seems like there is something not right about the usecase, which is leading to this solution. Personally, I would prefer not to bring this in, if the solution is to properly build OpenROAD for bazel (or whatever platform) instead of building on one platform and moving the binary around.

Reply:
It is not one or the other; there's no conflict here.

This PR is boarding up a broken window (lack of resources for CI + programming), longer term I hope to clean up bazel-orfs.

Reply:
How does @QuantamHD handle the OpenROAD build for his bazel setup? He must have encountered a similar issue

Reply:
I believe they use a monorepo and build from source.

Reply:
I also saw some patching of OpenROAD going on, so my short answer is that it would take a bit of time to investigate what they are doing.

In the future, I'd like to see an officially maintained bazel build of OpenROAD with an artifact repository that I can use, just like Docker images. Would save tons of building time for anyone involved + be single source of truth + better chance of everyone running the exact same version w.r.t. bug reports and communicating in the community...

Perhaps bazel_hdl could use it too?

That's not happening tomorrow though...